teanga/spacy/teanga_spacy.py

```diff
@@ -0,0 +1,107 @@
+try:
+    import spacy
+except ImportError:
+    raise ImportError("SpaCY is required for the SpaCY service")
+from teanga import Service
+
+class SpaCy(Service):
+    """A service that uses spaCy to tokenize and tag text.
+
+    This service requires a SpaCY model name as a parameter. The model
+    name is the name of a SpaCY model that has been installed on the
+    system. For example, the model "en_core_web_sm" is a small English
+    model that can be installed with the command:
+
+    python -m spacy download en_core_web_sm
+
```

Comment:
Unless the pipeline abstracts users away from it needing to enter models manually, then it would be helpful to link users to the spaCy page where they can find this and other models at this point: [https://spacy.io/models/en](url)

Reply:
Good idea!